test/test-server/utils.go

```diff
@@ -251,6 +252,12 @@ func writeResourceNotFoundError(w http.ResponseWriter) error {
 	return err
 }
 
+func writeFeedbackExceedsMaxLengthError(w http.ResponseWriter) error {
```

Comment:
Looks like this only gets used in one place, so we can move this into the function that uses it.